gitlab-examples/gitlab-pipline/gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+image: gradle:latest
+
+before_script:
+  # This variable is used to tell Gradle from which repository to download the build-info-extractor.jar. Should be server-id/repo-name
+  - JFROG_CLI_EXTRACTORS_REMOTE=$ARTIFACTORY_SERVER_ID/$ARTIFCATORY_REPO
```

Comment:
I think we should explain the motivation behind this environment variable and the instructions for configuring the remote repository. Actually, this is necessary only for air-gapped environments.
The following link contains all needed information: https://www.jfrog.com/confluence/display/CLI/CLI+for+JFrog+Artifactory#CLIforJFrogArtifactory-DownloadingtheMavenandGradleExtractorJARs
Maybe we should add it too.

Reply:
Regarding your last comment about Gradle Extractor Jar, do you think we should add the explanation, or remove it from the example?
I feel it is adding unnecessary complexity to the example, and removed the focus from the main point.
I guess common use case will be using GitLab with access to the internet.
P.S.
Tested again, and it seems that -b flag was not  required for artifactoryPublish task, so I removed it.

Reply:
Removing JFROG_CLI_EXTRACTORS_REMOTE from the example sounds good to me.
If you want, you can point in the README GitLab enterprise users without Internet users to the link.

Reply:
I do not think there is a need to add the link at this point. Let's leave it as it is.